ts-tests/tests/test-eth-balance.ts

```diff
@@ -145,7 +219,7 @@ async function get_assets(api: ApiPromise, alice: KeyringPair) {
   console.log(`Linked Ethereum balances of Alice are: ${assetsBalances.toString()}`);
   
   // TODO fetch real time balance and compare it here
```

Comment:
You mean compare the balance with a call of web3.getBalance?

Reply:
yes, that's the idea for next step.